### PR TITLE
Move RabbitMQ properties to a separate class

### PR DESCRIPTION
**Pull request by henning-roos**
### Description of the Change
RabbitMQ related properties now located in separate class RMQProperties.
Removed unnecessary tests that were duplicated.

### Benefits
Separation of concern: RMQProperties extracted from RMQHandler logic.

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: @Christoffer-Cortes @e-edling-ericsson @e-pettersson-ericsson @henning-roos @a-breid-ericsson @m-linner-ericsson @saif-ericsson @tobiasake

### CODE REVIEW

File: src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import com.ericsson.ei.erqueryservice.ERQueryService;
 import com.ericsson.ei.handlers.ObjectHandler;
 import com.ericsson.ei.handlers.RMQHandler;
```
Comment (m-linner-ericsson) [inline src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java]: I guess we don't need the import for RMQHandler now?

**Comment by tobiasake** [comment]
Can't we create a class that collects all Spring/Java properties,, so we get only one object api for all properties. This was an idea long time to implement, but has always ended with "that we can do later".

**Comment by e-edling-ericsson** [comment]
@tobiasake I don't quite agree that we should have one class for every property out there. Maybe one package though were all classes that reads properties exists. Otherwise it could be quite hard to know which properties the application requires without looking through every possible class in the project.

The reason I don't want them in the same big class is that it will be huge. I think it's good to divide it up "logically" into it's own property classes.

**Comment by tobiasake** [comment]
Well, you can structure properties class similar as it structured in application.properties file. I guess its ok to have in different classes in same package. But if a class need properties from different properties objects,, then it will be alot of Autowiring, which can be unnecessary from some users point of view or opinions, which prefer all properties was collected in same properties class and only one autowire properties object is required to access all properties.

File: src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java (hunk 1 above)
Comment (henning-roos) [reply src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java]: Done in upcoming commit

**Pull request merged**